stdlib/format.mli

```diff
@@ -1246,6 +1246,21 @@ val pp_print_text : formatter -> string -> unit
   @since 4.02
 *)
 
+val format_text: ('a,'b,'c,'d,'e,'f) format6 -> ('a,'b,'c,'d,'e,'f) format6
+(** [text fmt] replaces spaces and newlines in the format string literal [fmt]
+    with hint breaks or forced newlines:
+- Only ascii spaces ([" " = "\u{20}"]) and linefeeds ["\n" = "\u{A}"] are
+  replaced. In particular, this means that non-breaking spaces (for instance
+  ["\u{A0}"]) are left unchanged.
```

Comment:
I think it would be better to talk in `Format` speak: 

- Spaces `" "` (U+0020) are replaced by `"@ "`.
- Newlines `"\n"` (U+000A) are converted to `"@<0 1>"`.
- The rest is kept unchanged and in particular, non-breaking spaces like U+00A0.